iset.mm

```diff
@@ -61943,8 +61943,40 @@ the first case of his notation (simple exponentiation) and subscript it
     cvv fndm mpbid ) CADZABEZFZCGZCHIZACHIZUCCRECJZUEABCKUAUGUBACLMCRNOUAUEUFPU
     BUAUDACHACSQMT $.
 
+  ${
+    $d w y z A $.  $d w y z B $.
+    mapsnen.1 $e |- A e. _V $.
+    mapsnen.2 $e |- B e. _V $.
+    $( Set exponentiation to a singleton exponent is equinumerous to its base.
+       Exercise 4.43 of [Mendelson] p. 255.  (Contributed by NM, 17-Dec-2003.)
+       (Revised by Mario Carneiro, 15-Nov-2014.) $)
+    mapsnen $p |- ( A ^m { B } ) ~~ A $=
+      ( vz vw vy csn cmap cv cfv cop cvv wcel snex vex a1i wceq wa wex co fnmap
+      cxp wfn fnovex mp3an fvex opex mapsn abeq2i anbi1i r19.41v df-rex 3bitr2i
+      fveq1 syl6eq eqeq2d equcom syl6bb pm5.32i anbi2i anass ancom exbii eleq1w
+      wrex fvsn opeq2 sneqd anbi12d ceqsexv 3bitri en2i ) EFABHZIUAZABEJZKZBFJZ
+      LZHZIMMUCUDAMNVNMNVOMNUBCBDOAVNMMIUEUFCVQMNVPVONZBVPMMEPDUGQVTMNVRANZVSBV
+      RDFPZUHOQWAVRVQRZSZGJZANZVPBWFLZHZRZWDSZSZGTZWFVRRZWGWJSZSZGTWBVPVTRZSZWE
+      WJGAVFZWDSWKGAVFWMWAWSWDWSEVOGABECDUIUJUKWJWDGAULWKGAUMUNWLWPGWLWGWJWNSZS
+      WOWNSWPWKWTWGWJWDWNWJWDVRWFRWNWJVQWFVRWJVQBWIKWFBVPWIUOBWFDGPVGUPUQFGURUS
+      UTVAWGWJWNVBWOWNVCUNVDWOWRGVRWCWNWGWBWJWQGFAVEWNWIVTVPWNWHVSWFVRBVHVIUQVJ
+      VKVLVM $.
+  $}
+
   ${
     $d x y A $.  $d x y B $.  $d x y C $.  $d x y D $.  $d x y V $.
+    $( Set exponentiation: ordinal 1 to any set is equinumerous to ordinal 1.
+       Exercise 4.42(b) of [Mendelson] p. 255.  (Contributed by NM,
+       17-Dec-2003.) $)
+    map1 $p |- ( A e. V -> ( 1o ^m A ) ~~ 1o ) $=
+      ( vx vy wcel c1o cmap co c0 cxp cvv 1oex a1i cv 0ex p0ex wceq wa syl6rbb
```

Comment:
If ~mapsnen was provided in closed form ( ` ( ( A e. V /\ B e. W ) -> ( A ^m { B } ) ~~ A ) ` , ~map1 would follow immediatly applying ~1oex.

Reply:
GS provides the deduction form ~mapsnend of this theorem in his mathbox (in set.mm). This should be moved to main, and ~mapsnen and ~map1 can be derived from it afterwards (and maybe other proofs using ~mapsnen could be shortened). I'll do that in set.mm at least with my next PR.

Reply:
Why would `map1` follow from `mapsnen` ? The exponent in `map1` doesn't have to be a singleton.

Reply:
Ah, sorry, I always confuse the two operands for the ^m operation...
However, ~map1 can be generalized to a theorem similar to ~mapsnen:

snmapen $p |- ( ( A e. V /\ B e. W ) -> ( { A } ^m B ) ~~ { A } ) $=

The proof of this theorem is not much longer than the proof of ~map1, therefore I will provide this with my next PR (for set.mm).

Reply:
I like both theorems proposed by @avekens, except I would write the second as
```
snmapen $p |- ( ( A e. V /\ B e. W ) -> ( { A } ^m B ) ~~ 1o ) $=
```
since in the world of equinumerosity, the standard representant for "one" is 1o.